GoogleTestAdapter/Core/Runners/PreparingTestRunner.cs

```diff
@@ -11,6 +11,11 @@
 
 namespace GoogleTestAdapter.Runners
 {
+    public enum TestStatus 
+    {
+        TestSetup,
+        TestTeardown
+    }
     public class PreparingTestRunner : ITestRunner
     {
         public const string TestSetup = "Test setup";
```

Comment:
Let's move then to `TestExecutorTestsBase` then so we won't have unlocalized strings in this binary.